pkg/db/db.go

```diff
@@ -217,6 +220,33 @@ func (c *Client) initOCIArtifact(opt types.RegistryOptions) (*oci.Artifact, erro
 	return art, nil
 }
 
+func (c *Client) downloadDB(opt types.RegistryOptions, dst string) error {
+	downloadOpt := oci.DownloadOption{MediaType: dbMediaType}
+	if c.artifact != nil {
+		return c.artifact.Download(context.Background(), dst, downloadOpt)
+	}
+
+	for i, repo := range c.dbRepositories {
+		a, err := c.initOCIArtifact(repo, opt)
+		if err != nil {
+			return err
+		}
+
+		log.Info("Downloading vulnerability DB...", log.String("repo", repo.String()))
+		if err := a.Download(context.Background(), dst, downloadOpt); err != nil {
+			log.Error("Failed to download vulnerability DB",
+				log.String("repo", repo.String()), log.Err(err))
+			if i < len(c.dbRepositories)-1 {
+				log.Info("Trying to download vulnerability DB from other repository...")
+			}
+			continue
+		}
+		return nil
```

Comment:
Should we also have a debug log to say where the DB was downloaded from?

Reply:
It's logged https://github.com/aquasecurity/trivy/pull/7605/files#diff-5da3684806835246d262230050593f460b12b6c0e3966df174e6061be0e9e575R235

Reply:
I meant to say log the success of the DB. Something along the lines of "Successfully download from ... " as a debug print. But it's just a nit.

Reply:
Got it now, I added https://github.com/aquasecurity/trivy/pull/7605/commits/d75a5846290a44af6afe462b386efe179a298343